packages/rs-drive/src/lib.rs

```diff
@@ -55,5 +55,5 @@ pub mod state_transition_action;
 #[cfg(any(feature = "server", feature = "verify", feature = "fixtures-and-mocks"))]
 pub mod util;
 /// Contains a set of useful grovedb proof verification functions
-#[cfg(feature = "verify")]
+#[cfg(any(feature = "server", feature = "verify"))]
```

Comment:
@QuantumExplorer this is just monkey-pached, please fix correctly in your branch